### PR TITLE
Introduce depositOperations and redeemOperations getters

### PR DESCRIPTION
**Pull request by 0xrin1**
- Introduce depositOperations and redeemOperations getters
- Corresponding tests

### CODE REVIEW

File: test/strategy/anchor/AnchorUSTStrategy.spec.ts [hunk 1]
```diff
@@ -521,6 +521,32 @@ describe("AnchorUSTStrategy", () => {
 
       expect(await strategy.redeemOperationLength()).equal(2);
     });
+
+    it("Returns array of depositOperations when calling getter function", async () => {
```
Comment (ryuheimat) [inline test/strategy/anchor/AnchorUSTStrategy.spec.ts:525]: These cases are not related with `#initRedeemStable` function.
Can you make new `describe`?

Comment (0xrin1) [reply test/strategy/anchor/AnchorUSTStrategy.spec.ts:525]: makes sense!

I will kill the PR because of discussion had on slack

Next time I add tests I'll make sure that the `describe` block that they're in makes sense in terms of what the `it` cases do

thanks!
